download/download.go

```diff
@@ -7,8 +7,8 @@ import (
 	"time"
 
 	"github.com/rodrigo-brito/ninjabot/service"
-
```

Comment:
service is a internal package, can you merge the imports?

Reply:
Sure, done!